pneumonia-detection/README.md

```diff
@@ -0,0 +1,44 @@
+# Pneumonia Detection
+
+This is an example to demonstrate how to use [Deep Java Library](http://djl.ai) to detect Pneumonia based on Chest X-ray images.
+
+For more details, please follow [Chest X-Ray Images (Pneumonia)](https://www.kaggle.com/paultimothymooney/chest-xray-pneumonia) on Kaggle 
+and [this Kernel](https://www.kaggle.com/aakashnain/beating-everything-with-depthwise-convolution).
+
+You can obtain the training script from the above Kernel. We will use the model and DJL for prediction.
+
+## Setup
+run the following command to build the project:
+
+`./gradlew build`
+
+## Run prediction on images
+
+### Prepare images
+
+You can download some of the X-ray images for testing from [here](https://www.kaggle.com/paultimothymooney/chest-xray-pneumonia?).
```

Comment:
Remove ? and the end of url

Reply:
they can still download and test the test dataset